docs/articles/documentation/recipes/access-environment-variables-in-tests.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: docs
+title: Access Environment Variables in Tests
+permalink: /documentation/recipes/access-environment-variables-in-tests.html
+---
+# Access Environment Variables in Tests
+
+Use the Node.js [process.env](https://nodejs.org/api/process.html#process_process_env) property to access environment variables in test code.
+
+The following example prints the `PATH` envornment variable:
+
+```js
+fixture `My Fixture`;
+
+test('Print an Environment Variable', async t => {
```

Comment:
I guess you should before describe how to set an environment variable and after access to its value in the test.
For example:
## Set Environment Variables
...general information without TestCafe running...

## Access Environment Variables in Tests
The full example of the environment variable using 
```js
fixture `My Fixture`;
test('Print an Environment Variable', async t => { 
    console.log('Is in dev mode:' ', process.env.DEV_MODE);
});

set DEV_MODE=true
testcafe chrome test.js
```

Reply:
Well, I assumed that most people know how to set environment variables, so they just need to learn how to access them. And those few who don't know how to set them yet, will read further.

Okay, maybe this order is not intuitive.

But I don't get why we should first say how to set the env variables without TestCafe, then show how to access them, and then how to set them with TestCafe.

Reply:
So, I edited the topic - and now it first shows setting the variables including TestCafe launch, and then it says about accessing them.